Nondeterminism.md

```diff
@@ -31,6 +31,7 @@ currently admits nondeterminism:
    nondeterministic.
  * Out of bounds heap accesses *may* want
    [some flexibility](AstSemantics.md#out-of-bounds)
+ * The [`page_size` global constant](AstSemantics.md#resizing)
```

Comment:
Clarify that this is a size that's guaranteed to be allocatable, and that passing less than this to relevant functions cause them to trap.